front/app/containers/ProjectsShowPage/timeline/Timeline.tsx

```diff
@@ -219,22 +212,19 @@ const PhaseContainer = styled.div<{ width: number; breakpoint: number }>`
 interface Props {
   projectId: string;
   className?: string;
+  selectedPhase: IPhaseData;
+  setSelectedPhase: (phase: IPhaseData) => void;
 }
 
-const Timeline = memo<Props>(({ projectId, className }) => {
-  const locale = useLocale();
-  const currentTenant = useAppConfiguration();
+const Timeline = ({
+  projectId,
+  className,
+  selectedPhase,
+  setSelectedPhase,
+}: Props) => {
   const phases = usePhases(projectId);
-
-  const [selectedPhase, setSelectedPhase] = useState<IPhaseData | null>(null);
-
-  useEffect(() => {
-    const subscription = selectedPhase$.subscribe((selectedPhase) => {
-      setSelectedPhase(selectedPhase);
-    });
-
-    return () => subscription.unsubscribe();
-  }, []);
```

Comment:
Removed all of this and passed these values from the parent, so we're not managing this on two levels.